comment-dwim-2.el

```diff
@@ -60,8 +60,8 @@
 (require 'cl-lib)
 
 (defvar comment-dwim-2--inline-comment-behavior 'kill-comment
-  "Behavior of `comment-dwim-2' when it is being repeated and is
-encountering an inline comment.  Possible values are:
+  "Behavior of `comment-dwim-2' when repeated and at an inline comment.
```

Comment:
No, your version is better. I am not a native speaker and my English may be awkward sometimes :)